Headers/DebugServer2/Architecture/X86_64/CPUState.h

```diff
@@ -85,8 +86,18 @@ struct CPUState64 {
     uint16_t fstw;
     uint16_t fctw;
     uint16_t ftag;
-    uint32_t firip;
-    uint32_t forip;
+    union {
+      struct {
+        uint64_t firip;
+        uint64_t firdp;
+      } x86_64;
+      struct {
+        uint32_t fioff;
+        uint32_t fiseg;
+        uint32_t fooff;
+        uint32_t foseg;
+      } x86;
```

Comment:
It's more useful to expose as 4 separate registers instead of as 2 registers here. LLDB expects these 4 registers instead of firip and firdp anyway.